recipes-core/customizations-example/customizations-example_0.1-iot2050-debian.bb

```diff
@@ -19,9 +19,7 @@ DEBIAN_DEPENDS = "openssh-server, bluez, pulseaudio-module-bluetooth, network-ma
 SRC_URI = " \
     file://status-led.service \
     file://postinst \
-    file://board-configuration \
     file://board-configuration.service \
-    file://board-configuration.json \
```

Comment:
Looks inconsistent to me: board-configuration tool is factored out, but the file describing it as service is no? And no dependency is modeled between that package and this customization thing here?

Reply:
> Looks inconsistent to me: board-configuration tool is factored out, but the file describing it as service is no? And no dependency is modeled between that package and this customization thing here?

Put related resources together, pls review.